src/main/resources/static/js/thing.js

```diff
@@ -70,58 +70,58 @@ function createThing() {
     $.ajax({
         type: "POST",
         url: "thing/create",
-        datatype: 'json',
-        contentType: 'application/json',
+        datatype: "json",
+        contentType: "application/json",
         data: JSON.stringify(mydata),
         error: function (e) {
             $.notify({
-                message: 'Thing could not be created, check the Log for errors'
+                message: "Thing could not be created, check the Log for errors"
             }, {
                 allow_dismiss: true,
-                type: 'danger',
+                type: "danger",
                 placement: {
                     from: "top",
                     align: "left"
                 },
                 animate: {
-                    enter: 'animated fadeInDown',
-                    exit: 'animated fadeOutUp'
+                    enter: "animated fadeInDown",
+                    exit: "animated fadeOutUp"
                 },
                 z_index: 9000
             });
             addToLog(e.responseText);
         },
         success: function (e) {
             $.notify({
-                message: 'Thing created.'
+                message: "Thing created."
             },{
                 allow_dismiss:true,
-                type: 'info',
+                type: "info",
                 placement: {
                     from: "top",
                     align: "left"
                 },
                 animate: {
-                    enter: 'animated fadeInDown',
-                    exit: 'animated fadeOutUp'
+                    enter: "animated fadeInDown",
+                    exit: "animated fadeOutUp"
                 }
             });
-            addToLog('Thing created');
-            closeModal('dialog');
+            addToLog("Thing created");
+            closeModal("dialog");
 
-            var text = e.name + ' (' + e.frostId + ')';
+            var text = e.name + " (" + e.frostId + ")";
             var option = new Option(text, text, null, null);
-            option.setAttribute('data-value', JSON.stringify(e, null, 4));
+            option.setAttribute("data-value", JSON.stringify(e, null, 4));
 
-            $('#things').append(option).trigger('change');
-            $('#things').val(text).trigger('select2:select');
+            $("#things").append(option).trigger("change");
+            $("#things").val(text).trigger("select2:select");
         }
     });
 }
 
 function showLocationModal() {
-    var $modal = $('#thingfooter').find('button:eq(0)');
-    $modal.attr('onclick', 'createLocation()');
-    $modal.html('Create Location');
-    modal('thingdialog', 'location.html', null, 'Create a Location')
+    var $modal = $("#thingfooter").find("button:eq(0)");
+    $modal.attr("onclick", "createLocation()");
+    $modal.html("Create Location");
+    modal("thingdialog", "location.html", null, "Create a Location")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['modal' is not defined. (no-undef)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)